CHANGELOG.md

```diff
@@ -4,10 +4,12 @@ Mapbox welcomes participation and contributions from everyone.
 
 ## Unreleased
 #### Features
+- Added `ButtonCustomParams` to customise default Drop-In buttons. [#6327](https://github.com/mapbox/mapbox-navigation-android/pull/6327)
 #### Bug fixes and improvements
 - Optimized calls to modify route arrow layer visibility. [#6308](https://github.com/mapbox/mapbox-navigation-android/pull/6308)
 - Provide better java support for `MapboxNavigationApp`. [#6292](https://github.com/mapbox/mapbox-navigation-android/pull/6292)
 - Expose `PointAnnotationOptions` in Drop-In UI to allow users to define the destination marker icon and it's positioning. [#6314](https://github.com/mapbox/mapbox-navigation-android/pull/6314)
+- Fixed an issue where custom style of a Drop-In button wasn't handled. [#6327](https://github.com/mapbox/mapbox-navigation-android/pull/6327)
```

Comment:
Do we need 2 changelog statements? They both have similar  meaning to me. 
Suggestion:

`Introduced MapboxExtendableButtonParams to allow end users to customize the default shape of Drop-In UI buttons from rectangular to square, circle, oval and others.`